packages/components/src/dropdown-menu/index.js

```diff
@@ -67,7 +64,7 @@ function DropdownMenu( {
 						aria-haspopup="true"
 						aria-expanded={ isOpen }
 						label={ label }
-						labelPosition={ __unstableLabelPosition }
+						labelPosition={ position }
```

Comment:
I'm not sure about this change? Can you clarify? It seems that the position prop is meant to be used for something else?

Reply:
If I'd make it a stable api,  I'd probably name it `tooltipPosition`

Reply:
Yep, but in general, it aligns with where you would expect to see the tooltip :)

I can add `tooltipPosition` as well but maybe we should introduce `toggleProps` and `contentPropos` instead so you could have more control over them. The only issue I see here is that it's not easy to pass an object as a prop because it will cause re-renders every time `DropdownMenu` re-renders.

Reply:
I don't think the props align, I can see valid use-cases for showing a dropdown under the toggle but the tooltip on top of it. I don't have strong opinions on whether it should be a `toggleProps` prop or just a position one.

Reply:
@jorgefilipecosta added `popoverProps` prop described as an object for `Dropdown` components in #14867. There is a bit different handling for `Popover` component where all remaining props are passed down using rest operator:

https://github.com/WordPress/gutenberg/blob/master/packages/components/src/popover/index.js#L267

We probably should optimize APIs to go in one direction moving forward. In this particular case, it feels like using `toggleProps` would solve the issue we have and keep it more flexible for future use cases.

Reply:
> We probably should optimize APIs to go in one direction moving forward. In this particular case, it feels like using toggleProps would solve the issue we have and keep it more flexible for future use cases.

I think using a toggleProps object as referred seems like a good option.

Reply:
I introduced 3 new props based on the prior work with `Dropdown` component as implemented in #14867:
- `popoverProps`
- `toggleProps`
- `menuProps`

This allows us to deprecate two props which can be set using this pass-through objects and further simplify public API. For backward compatibility, they are still supported but they will warn on the JS console. I assumed they aren't used widely outside of Gutenberg.